requests-for-comments/2024-06-21-warehouse-launch-plan.md

```diff
@@ -0,0 +1,50 @@
+## [Marketing] Data warehouse launch plan
+
+The data warehouse has been in an opt-in beta for a while, and has now been opened up to a public open beta. It'll be at least a few weeks before it launches (we need to give current users two weeks notice of pricing changes) BUT that means it's the perfect time to plan. 
+
+### Ahead of launch
+
+- [ ] Publish a Stripe data warehouse tutorial - Marketing
+- [ ] Publish a Hubspot data warehouse tutorial - Marketing
+- [ ] Publish a Postgres data warehouse tutorial - Marketing
```

Comment:
Because Postgres is so flexible, it makes it really tricky to write a tutorial about anything specific
```suggestion
- [ ] Publish a [Stripe data warehouse tutorial](https://github.com/PostHog/posthog.com/issues/8662) - Marketing
- [ ] Publish a [Hubspot data warehouse tutorial](https://github.com/PostHog/posthog.com/issues/8664) - Marketing
- [ ] Publish a [Zendesk data warehouse tutorial](https://github.com/PostHog/posthog.com/issues/8663) - Marketing
```